### ISSUE TITLE
Quick API index page erroring on load

### ISSUE THREAD

**Issue by pvcraven** [pythonarcade/arcade#1312]
## Bug Report

Quick API index file generating error.

### Actual behavior:

Navigating to 2.6.15 quick index api generates "DataTables warning: Non-table node initialisation (DIV). For more information about this error, please see http://datatables.net/tn/2"

### Expected behavior:

No error

### Steps to reproduce/example code:

Go to URL: https://api.arcade.academy/en/latest/quick_index.html

**Comment by pvcraven** [comment]
I think a fix for this exists in 2.7 already.

**Comment by pvcraven** [comment]
Issue is we have two elements with the same id:
![Screen Shot 2022-08-26 at 3 44 24 PM](https://user-images.githubusercontent.com/1463116/186988129-6a4c2f04-938b-4416-af41-01c221b3e97a.png)

This is generated from:

```rst
.. list-table::
   :widths: 50 50
   :header-rows: 1
   :name: quickapi
   :class: display
```
Maybe this is Furo or Sphinx not doing a proper generation of elements?

**Comment by pvcraven** [comment]
Should be fixed in 2.7 branch.

**Comment by pvcraven** [comment]
Fix pushed to prod, closing.
